eval.go

```diff
@@ -339,16 +348,20 @@ nextFileMatch:
 		repoMatchCount += len(fileMatch.LineMatches)
 		repoMatchCount += matchedChunkRanges
 
-		if opts.DebugScore {
-			fileMatch.Debug = fmt.Sprintf("score:%.2f <- %s", fileMatch.Score, fileMatch.Debug)
-		}
```

Comment:
why did you remove this?

Reply:
I just moved it to the respective scoring function, see `scoreFile` or `scoreFilesUsingBM25`.